Extensions/IISWebAppDeploy/Src/Tasks/SqlDacpacDeploy/SqlPackageOnTargetMachines.ps1

```diff
@@ -376,7 +376,7 @@ function Get-SqlPackageCmdArgs
             $sqlPkgCmdArgs = [string]::Format('{0} /TargetDatabaseName:"{1}"', $sqlPkgCmdArgs, $databaseName)
         }
 
-        if($authscheme -eq "SQL Server Authentication")
+        if($authscheme -eq "sqlServerAuthentication")
```

Comment:
Modify tests that are getting affected as well. You can verify this by running gulp test